pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -114,6 +115,14 @@ stdenv.mkDerivation rec {
       substituteInPlace $i \
         --replace /usr/bin $out/bin
     done
+
+    rm -f $out/opt/kingsoft/wps-office/office6/libfreetype.so.6
+    ln -sf ${freetype.overrideAttrs (e: rec {
+      patches = e.patches ++ [
+        ./0000-WPS-compatiblity.patch
```

Comment:
I'm not sure whether nixpkgs accepts this patch，wps is not adapted to freetype>2.13.0（need [this patch](https://gitlab.freedesktop.org/freetype/freetype/-/merge_requests/304/))， @ttuegel

Reply:
If a patch to FreeType only affects WPS software, the patch is acceptable, 
e.g. by keeping the FreeType variant only in the WPS directory. 

There are also patches in Archlinux, check out this https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=freetype2-wps